sql-plugin/src/main/scala/org/apache/spark/sql/rapids/arithmetic.scala

```diff
@@ -46,12 +46,14 @@ object GpuAnsi extends Arm {
 
   def assertMinValueOverflow(cv: GpuColumnVector, op: String): Unit = {
     withResource(minValueScalar(cv.dataType())) { minVal =>
-      withResource(cv.getBase.equalToNullAware(minVal)) { isMinVal =>
-        withResource(isMinVal.any()) { anyFound =>
-          if (anyFound.isValid && anyFound.getBoolean) {
-            throw new ArithmeticException(s"One or more rows overflow for $op operation.")
-          }
-        }
+      assertMinValueOverflowImp(minVal, cv, op)
+    }
+  }
+
+  def assertMinValueOverflowImp(minVal: Scalar, cv: GpuColumnVector, op: String): Unit = {
```

Comment:
Nit: I don't see the need for `Imp` in the name here.  Normally `Imp` or `Impl` is designed for private implementations of an interface, but this is public and called which defeats the purpose.  This should either just rely on an overload of the original name or have a more descriptive distinction than just `Imp`.

Reply:
Updated